docs/Archive/index.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I think you should move this into `archive/`

Reply:
It already is, right?

Reply:
No, it's in `Archive/`, while the rest of the archived files are in `archive/`

Reply:
Sorry @tsalo - I'm not 100% sure what you mean here?

Reply:
Ahh! I renamed archive --> Archive

Reply:
I would do the opposite

Reply:
There are two folders: `archive` (lower-case "a") and `Archive` (upper-case "A"). The latter is where your file `index.md` is located, but the former is where most of the other archived files are located. I would recommend moving your `index.md` from `Archive` to `archive`, so that everything is in one place.

Reply:
under docs? I only see Archive

Reply:
https://github.com/PennLINC/PennLINC.github.io/tree/main/docs/archive

Reply:
I see that in the tree, but for some reason locally I don't see "archive" - I can try renaming Archive --> archive